scripts/README.md

```diff
@@ -0,0 +1,157 @@
+](https://github.com/github/scripts-to-rule-them-all) was used as a template. They were tested using Ubuntu 18.04.3 LTS on Windows 10 and using GridOS 26, a derivative of Red Hat Fedora 26, on the Lincoln Laboratory Supercomputing Cluster.
+
+- [Scripts](#scripts)
+  - [`FLOOD_ANALYSIS_CORE` and Execution](#flood_analysis_core-and-execution)
+  - [Dependencies](#dependencies)
+    - [Linux Shell](#linux-shell)
+    - [Proxy and Internet Access](#proxy-and-internet-access)
+    - [Superuser Access](#superuser-access)
+  - [The Scripts](#the-scripts)
+    - [script/bootstrap](#scriptbootstrap)
+      - [Packages](#packages)
+      - [Git Submodules](#git-submodules)
+    - [script/setup](#scriptsetup)
+      - [Data](#data)
+
+## `FLOOD_ANALYSIS_CORE` and Execution
+
+These scripts assume that `FLOOD_ANALYSIS_CORE` has been set. Refer to the repository root [README](../README.md) for instructions.
+
+## Dependencies
+
+### Linux Shell
+
+The scripts need to be run in a Linux shell. For Windows 10 users, you can use [Ubuntu on Windows](https://tutorials.ubuntu.com/tutorial/tutorial-ubuntu-on-windows#0). Specifically for Windows users, system drive and other connected drives are exposed in the `/mnt/` directory. For example, you can access the Windows C: drive via `cd /mnt/c`.
+
+If you modify these scripts, please follow the [convention guide](https://github.com/Airspace-Encounter-Models/em-overview/blob/master/CONTRIBUTING.md#convention-guide) that specifies an end of line character of `LF (\n)`. If the end of line character is changed to `CRLF (\r)`, you will get an error like this:
+
+### Proxy and Internet Access
+
+The scripts will download data using [`curl`](https://curl.haxx.se/docs/manpage.html) and [`wget`](https://manpages.ubuntu.com/manpages/trusty/man1/wget.1.html), which depending on your security policy may require a proxy.
+
+The scripts assume that the `http_proxy` and `https_proxy` linux environments variables have been set.
+
+```bash
+export http_proxy=proxy.mycompany:port
+export https_proxy=proxy.mycompany:port
+```
+
+You may also need to [configure git to use a proxy](https://stackoverflow.com/q/16067534). This information is stored in `.gitconfig`, for example:
+
+```git
+[http]
+	proxy = http://proxy.mycompany:port
+[https]
+	proxy = http://proxy.mycompany:port
+```
+
+### Superuser Access
+
+Depending on your security policy, you may need to run some scripts as a superuser or another user. These scripts have been tested using [`sudo`](https://manpages.ubuntu.com/manpages/disco/en/man8/sudo.8.html). Depending on how you set up the system variable, `FLOOD_ANALYSIS_CORE` you may need to call [sudo with the `-E` flag](https://stackoverflow.com/a/8633575/363829), preserve env.
+
+If running without administrator or sudo access, try running these scripts using `bash`, such as 
+
+```bash
+bash ./setup.sh
+```
```

Comment:
This is unclear.

Reply:
You can either execute a script directly (e.g. `./setup.sh` or within a specific shell environment (`bash ./setup.sh`). On enterprise systems, user permissions can vary across environments. On your personal computer, you're less likely to encounter situations like this.